src/rollback_interface.cpp

```diff
@@ -65,6 +65,7 @@ std::string RollbackAction::toString() const
 		os << ", " << itos(n_new.param2);
 		os << ", " << serializeJsonString(n_new.meta);
 		os << ')';
+		break;
```

Comment:
This is different behavior now. Is that intentional? If so it was bug before.

Reply:
It was an obvious bug.
Technically the behaviour of `l_rollback_revert_actions_by` changes a little, but I doubt mods will break due to this.